clients/drcachesim/tracer/output.cpp

```diff
@@ -138,7 +138,8 @@ reached_traced_instrs_threshold(void *drcontext)
     tracing_window.fetch_add(1, std::memory_order_release);
     // We delay creating a new ouput dir until tracing is enabled again, to avoid
     // an empty final dir.
-    DR_ASSERT(tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_TRACE);
+    DR_ASSERT(tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_TRACE ||
+              tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_L0_FILTER);
```

Comment:
See above about avoiding the duplicate load-acquire.  Ditto below for the multiple instances of this pattern.

Reply:
Done.